packages/apps/lightning-ui-docs/.storybook/addons/decorators/withLightning.js

```diff
@@ -21,7 +21,7 @@ import {
   context,
   utils,
   TextBox
-} from '@lightningjs/ui-components';
+} from '@lightningjs/ui-components/src';
```

Comment:
is this src addition really necessary?

Reply:
I believe so, but will need to confirm with @joshhowenstine

Reply:
Was this ever investigated? This seems fine for me locally without as all of those things should be exported by ui-components already.